extension/extension.cpp

```diff
@@ -342,8 +342,15 @@ bool Connect::SDK_OnLoad(char *error, size_t maxlen, bool late)
 		return false;
 	}
 
+	int steam3ServerFuncOffset = 0;

```

Comment:
should be intptr_t ☝🤓

Reply:
I'm following the code convention ☝🤓
https://github.com/asherkin/connect/blob/0c6274058e718d142189d331aae9b288b351360a/extension/extension.cpp#L149-L150

Reply:
Admittedly those ones are used as array indices, but I'm happy with either.